docs/features/software-catalog/catalog-customization.md

```diff
@@ -698,3 +698,36 @@ filter:
       target:
         $in: [group:default/admins, group:default/viewers]
 ```
+
+### Catalog Content Header
+
+The catalog page comes with a content header containing default elements, but you may want to add your own elements to it.
```

Comment:
I think adding an image here would help as it's not clear what this will all let you customize. This naming - `CatalogContentHeaderItemBlueprint` - makes me think it will replace the whole header but I'm not sure that's actually what it does.